build/checks.mk

```diff
@@ -1,10 +0,0 @@
-
```

Comment:
As far as I understand this change was reverted in https://github.com/particle-iot/device-os/pull/1664. I'd remove https://github.com/particle-iot/device-os/pull/1665/commits/f77ae441d6250b0a91371adba6739aff8c1b86d4 from this branch altogether in order to not accidentally overwrite #1664.